eventarc_gke/eventarc/main.tf

```diff
@@ -0,0 +1,82 @@
+/**
+
+* Copyright 2022 Google LLC
```

Comment:
Please change copyright to 2023 globally

Reply:
Done